src/fu-device.c

```diff
@@ -1904,10 +1925,14 @@ fu_device_incorporate (FuDevice *self, FuDevice *donor)
 	FuDevicePrivate *priv = GET_PRIVATE (self);
 	FuDevicePrivate *priv_donor = GET_PRIVATE (donor);
 	GPtrArray *parent_guids = fu_device_get_parent_guids (donor);
+	g_autoptr(FuMutexLocker) locker3 = fu_mutex_read_locker_new (priv_donor->metadata_mutex);
+	g_autoptr(FuMutexLocker) locker4 = fu_mutex_read_locker_new (priv_donor->parent_guids_mutex);
```

Comment:
Don't really like just adding on numeric lockers here.  `donor_metadata_locker` and `donor_parent_locker` maybe?

Reply:
Right, sorry, this was forgotten about.